src/main/java/tterrag/wailaplugins/plugins/PluginForestry.java

```diff
@@ -55,8 +50,14 @@
 import forestry.core.tiles.TileForestry;
 import forestry.core.utils.StringUtil;
 import forestry.plugins.PluginApiculture;
+import lombok.SneakyThrows;
+import magicbees.tileentity.TileEntityMagicApiary;
+import mcp.mobius.waila.api.IWailaDataAccessor;
+import mcp.mobius.waila.api.IWailaRegistrar;
+import mcp.mobius.waila.api.SpecialChars;
+import tterrag.wailaplugins.api.Plugin;
 
-@Plugin(name = "Forestry", deps = "Forestry")
+@Plugin(name = "Forestry", deps = { "Forestry", "MagicBees" })
```

Comment:
It would be better to add it as an optional dep, so the mod loads without Magic Bees